### PR TITLE
When clicking on the reset board you are now asked if you are sure

### LINKED ISSUE #1084: Ask for confirmation when board factory reset is requested
**Is your feature request related to a problem? Please describe.**
Mobiflight currently does not request user confirmation for factory resetting a board.

**Describe the solution you'd like**
Dialog requesting user confirmation before proceeding with factory reset.

**Describe alternatives you've considered**
There is no alternative.  If user accidentally presses the factory reset option in the menu, the board is immediately reset.

**Additional context**
Most lesser impact operations like uploading a device config request for confirmation.

### PR DESCRIPTION
**Pull request by Pitj3**
Fixes #1084 

Added dialog box asking for confirmation on resetting the board.
Used localization for EN and DE messages.

### CODE REVIEW

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
Thanks!

The message should read differently, it is not about config, you are resetting your board back to standard arduino and you will lose all information stored on it.

`Do you really want to uninstall the MobiFlight firmware? 
All your data will be removed from the board.`

I also believe that I switched to "Du" in the new German dialog messages.

`Möchtest Du wirklich die MobiFlight Firmware deinstallieren?
Alle Deine Daten werden vom Board gelöscht.`

**Comment by Pitj3** [comment]
Changes have been made as requested

**Comment by DocMoebiuz** [comment]
Thanks!
Please always mention the issue(s) that this PR "fixes" or "relates to" or "is part of"

With "fixes #..." the according issue will be set to "closed" automatically once the PR is merged.

Please also mention a short outline of the introduced changes, or touched ares, e.g.:

- [ ] Added a new message box
- [ ] i18n - added EN and DE messages

File: ProjectMessages/ProjectMessages.resx [hunk 1]
```diff
@@ -559,4 +559,10 @@ Check log for more details.</value>
   <data name="uiMessageValidationMustBeNumber" xml:space="preserve">
     <value>The value must be a valid number.</value>
   </data>
+  <data name="uiMessageResetConfirm" xml:space="preserve">
+    <value>Do you really want to uninstall the MobiFlight firmware?  All your data will be removed from the board.</value>
```
Comment (DocMoebiuz) [inline ProjectMessages/ProjectMessages.resx:563]: Tiny little typo. two spaces.

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
see my last comment and the typo here.

**Review by DocMoebiuz — APPROVED** [review]

**Pull request merged**